component/main.jsonnet

```
@@ -8,44 +8,71 @@ local params = inv.parameters.crossplane;
 local on_openshift4 = inv.parameters.facts.distribution == 'openshift4';
 local has_service_account(provider) = std.count(std.objectFields(params.serviceAccounts), provider) > 0;
 local has_any_service = std.length(std.filter(function(x) std.objectHas(params.serviceAccounts, x), std.objectFields(params.providers))) > 0;
+local missing_controller(provider_name) = std.count(std.objectFields(params.controllerConfigs), provider_name) == 0;
```

Just use `std.objectHas()` (https://jsonnet.org/ref/stdlib.html#objectHas) to check presence (or absence) of a field:

```suggestion
local missing_controller(provider_name) = !std.objectHas(params.controllerConfigs, provider_name);
```